docs/pages/database-access/guides/sql-server-ad.mdx

```diff
@@ -0,0 +1,315 @@
+---
+title: Database Access with Microsoft SQL Server with Active Directory authentication (Preview)
+description: How to configure Teleport Database Access with Microsoft SQL Server with Active Directory authentication.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="9.0"
+>
+  Database access for Microsoft SQL Server with Active Directory authentication
+  is available starting from Teleport `9.0`.
+</Details>
+
+<Admonition type="warning" title="Preview">
+  Database access for Microsoft SQL Server is currently in a Preview mode and
+  does not include audit logging of database query activity.
+</Admonition>
+
+This guide will help you to:
+
+- Install and configure Teleport.
+- Set up access to your SQL Server using Active Directory authentication.
+- Connect to the database server through Teleport.
+
+This guide will focus on Amazon RDS for SQL Server using AWS-managed Active
+Directory authentication.
+
+## Prerequisites
+
+- Teleport version >= `9.0`.
+- A SQL Server database with Active Directory authentication enabled.
+- A Windows machine joined to the same Active Directory domain as the database.
+- A Linux node joined to the same Active Directory domain as the database. This
+  guide will walk you through the joining steps if you don't have one.
+
+## Step 1/7. Set up Teleport Auth and Proxy
```

Comment:
Does this guide support Teleport Cloud? If so, we may want to remove this section and list "A running Teleport Auth and Proxy Service, whether in the Cloud or self hosted" (or similar) in the prerequisites.

Reply:
It would be a bit inconsistent with other guides that try and explain how to install Teleport as well (and avoid having to send users around between different guides). I've added an admonition explaining that this step is not needed for Cloud users - would that work?

Reply:
Now that we've merged [this commit](https://github.com/gravitational/teleport/commit/0499c6ef8e33d45b536521983198e3476a4d13c2#diff-b95c20e85ebd3429ca8c4a2f99ec113280dd50b784f4f3ff972f4c3f39a304a3), I think it's okay to remove this Admonition and keep this section, since it includes a tab for Cloud users.